concepts/eventing-code-guidelines.md

```diff
@@ -389,6 +389,64 @@ MarkReady(apiRuleNew) // instead consider using another function that explicitly
 ```
 </details>
 
+### Logging Guidelines
+
+Here are some general rules to standardize logs across eventing components:
```

Comment:
I think it's common to capitalize "Eventing" because it's a Kyma component. Correct me if I'm misunderstanding this here.

```suggestion
Standardize logs across Eventing components according to the following general and code-specific rules:
```

Reply:
Not sure, what you mean here. In the sentence I meant, that the logs should be capitalised, not the Kyma components.

Reply:
Not sure, what you mean here. In the sentence I meant, that the logs should be capitalised, not the Kyma components.

Reply:
I meant the word "Eventing" is usually written as `Eventing` not `eventing` when we refer to the Kyma component that Tunas take care of. 
Whereas, if it's the generic computer science term "eventing" about actions/occurrences recognized by a software, you can keep it in lower case, `eventing`, and you can disregard my suggestion to write `Eventing` in such cases.

Reply:
Ok, now I get what you mean. I replaced the sentence, because I like your suggestion more than mine.